docs/source/netutils/utilities/index.rst

```diff
@@ -0,0 +1,65 @@
+*******************
+Netutils Utilities
+*******************
+
+Netutils to Jinja2 Filters
+============================
+
+In an effort to simplify the process of adding netutil's functions to jinja2 as filters we have created a convenience function. Let me give you an example of one use case.
+Here is the current folder structure.
+
+.. code-block:: python
+
+    .
+    ├── p1.py
+    └── templates
+    └── test.j2
+
+Below is the code in the `test.j2` file.
+
+.. code-block:: jinja
+
+    IP Address + 200 = {{ "192.168.0.1/10" | ip_addition(200) }}
+
+Below is a code in the `p1.py` folder.
+
+.. code-block:: python
+
+    from jinja2.loaders import FileSystemLoader, PackageLoader
+    from jinja2 import Environment, PackageLoader, select_autoescape
+    from netutils.utils import jinja2_convenience_function
+
+    env = Environment(
+        loader=FileSystemLoader("templates"),
+        autoescape=select_autoescape()
+    )
+
+    for function_name, function_object in jinja2_convenience_function().items():
```

Comment:
I said extend, but I meant to say update, does this work? env.filters.update(jinja2_convenience_function())

Reply:
That does indeed work! Makes sense.